spec/features/categories_index_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+base_url = 'http://localhost:3000'
+
+RSpec.feature 'Categories#index view', type: :feature do
+  before(:each) do
+    @current_user = User.first
+    @current_user.confirm if @current_user.confirmed_at.nil?
+    @category = Category.find_by(user_id: @current_user.id)
+    visit "#{base_url}/categories"
+    fill_in 'Email', with: @current_user.email
+    fill_in 'Password', with: '123456'
+    click_button 'Log in'
+  end
+
+  scenario 'Displaying category names' do
+    visit "#{base_url}/categories"
+    expect(page).to have_content(@category.name, wait: 5)
+
+    expect(body).to have_content(@category.name)
+  end
+
+  scenario 'Displaying category records total amount' do
+    visit "#{base_url}/categories"
+    total = 0
+    expect(page).to have_content(total, wait: 5)
+    expect(body).to have_content(total)
+  end
```

Comment:
- [ ] some of the tests are failing please make sure to fix them so they are passing 
<img width="980" alt="Screenshot 2023-03-23 at 12 28 46" src="https://user-images.githubusercontent.com/61790587/227204124-2f24ca32-7aed-4055-ad36-9e7d613e96a2.png">